CONTRIBUTION.md

```diff
@@ -19,8 +19,8 @@ If any part of the project has bugs or documentation mistakes, please let us kno
 
 This is an outline of what a contributor's workflow looks like:
 
-- Fork the repository on GitHub
-- Clone the forked repository in your go path.
+- Fork the repository on GitHub.
+- Clone the forked repository outside your go path or refer to [kubernetes example](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#2-clone-fork-to-local-storage).
```

Comment:
Please add a comma here: ` outside your go path, or`.
Please change `refer to kubernetes example` to `the Kubernetes repo's example`. Since right now its not proper grammar, and we also want to capitalize Kubernetes according to their style guide.